### ISSUE TITLE
Panic while searching

### ISSUE THREAD

**Issue by fhanrath** [helix-editor/helix#6161]
### Summary

When I'm switching between files via mouse and I forgot to close a search dialog in one file, Helix often crashes when I try to search in the next file.

While this is not a show stopper for me, I'm able to reproduce it quite consistently on multiple machines (All Linux).

### Reproduction Steps

I tried this:

1. `hx`
2. `ctrl + f` to open file
3. `ctrl + w + v` split vertically
4. `ctr + f` open another file
5. switch to first file by mouse
6. `/` to search something (just type anything)
7. don't press Enter, switch to second file by mouse
8. `/` to search in second file

I expected this to happen:
The search dialog opens

Instead, this happened:
`thread 'main' panicked at 'range start index 159 out of range for slice of length 158', /build/.cargo/registry/src/github.com-1ecc6299db9ec823/regex-1.7.0/src/re_unicode.rs:653:9`

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

2023-03-02T21:29:51.929 helix_loader [DEBUG] Located configuration folders: ["/home/fenris/dotfiles/.helix"]
2023-03-02T21:29:51.934 helix_view::clipboard [INFO] Using wl-copy+wl-paste to interact with the system and selection (primary) clipboard
2023-03-02T21:29:51.934 helix_view::document [DEBUG] id 1 modified - last saved: 0, current: 0
2023-03-02T21:29:52.336 helix_term::application [DEBUG] received editor event: IdleTimer
2023-03-02T21:29:52.592 helix_view::document [DEBUG] id 1 modified - last saved: 0, current: 0
2023-03-02T21:29:52.736 globset [DEBUG] glob converted to regex: Glob { glob: "*.{zip,gz,bz2,zst,lzo,sz,tgz,tbz2,lz,lz4,lzma,lzo,z,Z,xz,7z,rar,cab}", re: "(?-u)^[^/]*\\.(cab|rar|7z|xz|Z|z|lzo|lzma|lz4|lz|tbz2|tgz|sz|lzo|zst|bz2|gz|zip)$", opts: GlobOptions { case_insensitive: false, literal_separator: true, backslash_escape: true }, tokens: Tokens([ZeroOrMore, Literal('.'), Alternates([Tokens([Literal('c'), Literal('a'), Literal('b')]), Tokens([Literal('r'), Literal('a'), Literal('r')]), Tokens([Literal('7'), Literal('z')]), Tokens([Literal('x'), Literal('z')]), Tokens([Literal('Z')]), Tokens([Literal('z')]), Tokens([Literal('l'), Literal('z'), Literal('o')]), Tokens([Literal('l'), Literal('z'), Literal('m'), Literal('a')]), Tokens([Literal('l'), Literal('z'), Literal('4')]), Tokens([Literal('l'), Literal('z')]), Tokens([Literal('t'), Literal('b'), Literal('z'), Literal('2')]), Tokens([Literal('t'), Literal('g'), Literal('z')]), Tokens([Literal('s'), Literal('z')]), Tokens([Literal('l'), Literal('z'), Literal('o')]), Tokens([Literal('z'), Literal('s'), Literal('t')]), Tokens([Literal('b'), Literal('z'), Literal('2')]), Tokens([Literal('g'), Literal('z')]), Tokens([Literal('z'), Literal('i'), Literal('p')])])]) }
2023-03-02T21:29:52.736 globset [DEBUG] built glob set; 0 literals, 0 basenames, 0 extensions, 0 prefixes, 0 suffixes, 0 required extensions, 1 regexes
2023-03-02T21:29:52.737 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/.git: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:29:52.738 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/roles/paru/.travis.yml: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:29:52.740 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/roles/nginx_reverse_proxy/templates/.env: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:29:52.740 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/roles/arch_base/templates/.gitconfig: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:29:52.740 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/roles/arch_base/.travis.yml: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:29:52.740 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/roles/podman/.travis.yml: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:29:52.741 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/roles/homeassistant/.travis.yml: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:29:52.741 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/roles/helix/.travis.yml: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:29:52.741 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/vscodium/.config: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:29:52.741 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/rofi/.config: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:29:52.741 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/bat/.config: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:29:52.741 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/zsh/.config: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:29:52.741 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/zsh/.zshrc: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:29:52.742 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/gtk/.config: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:29:52.742 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/gtk/.gtkrc-2.0: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:29:52.742 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/qt/.config: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:29:52.742 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/git/.gitconfig: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:29:52.742 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/wm/.config: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:29:52.742 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/helix/.config: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:29:52.742 helix_term::ui [DEBUG] file_picker init 6.032529ms
2023-03-02T21:29:52.742 helix_view::document [DEBUG] id 1 modified - last saved: 0, current: 0
2023-03-02T21:29:53.134 helix_view::document [DEBUG] id 1 modified - last saved: 0, current: 0
2023-03-02T21:29:53.284 helix_view::document [DEBUG] id 1 modified - last saved: 0, current: 0
2023-03-02T21:29:53.434 helix_view::document [DEBUG] id 1 modified - last saved: 0, current: 0
2023-03-02T21:29:53.584 helix_view::document [DEBUG] id 1 modified - last saved: 0, current: 0
2023-03-02T21:29:53.734 helix_view::document [DEBUG] id 1 modified - last saved: 0, current: 0
2023-03-02T21:29:53.899 helix_view::document [DEBUG] id 1 modified - last saved: 0, current: 0
2023-03-02T21:29:54.300 helix_term::application [DEBUG] received editor event: IdleTimer
2023-03-02T21:29:54.306 helix_view::document [DEBUG] id 1 modified - last saved: 0, current: 0
2023-03-02T21:29:55.114 helix_lsp::transport [INFO] -> {"jsonrpc":"2.0","method":"initialize","params":{"capabilities":{"textDocument":{"codeAction":{"codeActionLiteralSupport":{"codeActionKind":{"valueSet":["","quickfix","refactor","refactor.extract","refactor.inline","refactor.rewrite","source","source.organizeImports"]}}},"completion":{"completionItem":{"insertReplaceSupport":true,"resolveSupport":{"properties":["documentation","detail","additionalTextEdits"]},"snippetSupport":false},"completionItemKind":{}},"hover":{"contentFormat":["markdown"]},"publishDiagnostics":{},"rename":{"dynamicRegistration":false,"honorsChangeAnnotations":false,"prepareSupport":false},"signatureHelp":{"signatureInformation":{"activeParameterSupport":true,"documentationFormat":["markdown"],"parameterInformation":{"labelOffsetSupport":true}}}},"window":{"workDoneProgress":true},"workspace":{"applyEdit":true,"configuration":true,"didChangeConfiguration":{"dynamicRegistration":false},"executeCommand":{"dynamicRegistration":false},"symbol":{"dynamicRegistration":false},"workspaceFolders":true}},"processId":238229,"rootPath":"/home/fenris/dotfiles","rootUri":"file:///home/fenris/dotfiles","workspaceFolders":[{"name":"dotfiles","uri":"file:///home/fenris/dotfiles"}]},"id":0}
2023-03-02T21:29:55.120 helix_view::document [DEBUG] id 1 modified - last saved: 0, current: 0
2023-03-02T21:29:55.120 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:29:55.385 helix_lsp::transport [INFO] <- {"jsonrpc":"2.0","id":0,"result":{"capabilities":{"textDocumentSync":1,"completionProvider":{"resolveProvider":true,"triggerCharacters":["$","{"]},"hoverProvider":true,"documentHighlightProvider":true,"definitionProvider":true,"documentSymbolProvider":true,"workspaceSymbolProvider":true,"referencesProvider":true,"codeActionProvider":{"codeActionKinds":["quickfix"],"resolveProvider":false,"workDoneProgress":false}}}}
2023-03-02T21:29:55.385 helix_lsp::transport [INFO] <- {"capabilities":{"codeActionProvider":{"codeActionKinds":["quickfix"],"resolveProvider":false,"workDoneProgress":false},"completionProvider":{"resolveProvider":true,"triggerCharacters":["$","{"]},"definitionProvider":true,"documentHighlightProvider":true,"documentSymbolProvider":true,"hoverProvider":true,"referencesProvider":true,"textDocumentSync":1,"workspaceSymbolProvider":true}}
2023-03-02T21:29:55.385 helix_lsp::transport [INFO] -> {"jsonrpc":"2.0","method":"initialized","params":{}}
2023-03-02T21:29:55.385 helix_term::application [DEBUG] received editor event: LanguageServerMessage((0, Notification(Notification { jsonrpc: None, method: "initialized", params: None })))
2023-03-02T21:29:55.385 helix_lsp::transport [INFO] -> {"jsonrpc":"2.0","method":"textDocument/didOpen","params":{"textDocument":{"languageId":"bash","text":"#!/bin/bash\ngopass show -o fenris-admin/ansible/dotfiles_vault\n","uri":"file:///home/fenris/dotfiles/bin/password.sh","version":0}}}
2023-03-02T21:29:55.385 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:29:55.386 helix_lsp::transport [INFO] <- {"jsonrpc":"2.0","id":0,"method":"client/registerCapability","params":{"registrations":[{"id":"6a100d36-23b3-4d5c-bd6a-2de28331e750","method":"workspace/didChangeConfiguration","registerOptions":{"section":"bashIde"}}]}}
2023-03-02T21:29:55.386 helix_term::application [DEBUG] received editor event: LanguageServerMessage((0, MethodCall(MethodCall { jsonrpc: Some(V2), method: "client/registerCapability", params: Map({"registrations": Array [Object {"id": String("6a100d36-23b3-4d5c-bd6a-2de28331e750"), "method": String("workspace/didChangeConfiguration"), "registerOptions": Object {"section": String("bashIde")}}]}), id: Num(0) })))
2023-03-02T21:29:55.386 helix_term::application [ERROR] Language Server: Method not found client/registerCapability
2023-03-02T21:29:55.386 helix_lsp::transport [INFO] <- {"jsonrpc":"2.0","id":1,"method":"workspace/configuration","params":{"items":[{"section":"bashIde"}]}}
2023-03-02T21:29:55.386 helix_term::application [DEBUG] received editor event: LanguageServerMessage((0, MethodCall(MethodCall { jsonrpc: Some(V2), method: "workspace/configuration", params: Map({"items": Array [Object {"section": String("bashIde")}]}), id: Num(1) })))
2023-03-02T21:29:55.386 helix_lsp::transport [INFO] -> {"jsonrpc":"2.0","result":[null],"id":1}
2023-03-02T21:29:55.394 helix_lsp::transport [INFO] <- {"jsonrpc":"2.0","method":"window/logMessage","params":{"type":3,"message":"20:29:55.388 INFO BackgroundAnalysis: resolving glob \"**/*@(.sh|.inc|.bash|.command)\" inside \"file:///home/fenris/dotfiles\"..."}}
2023-03-02T21:29:55.394 helix_term::application [DEBUG] received editor event: LanguageServerMessage((0, Notification(Notification { jsonrpc: Some(V2), method: "window/logMessage", params: Map({"message": String("20:29:55.388 INFO BackgroundAnalysis: resolving glob \"**/*@(.sh|.inc|.bash|.command)\" inside \"file:///home/fenris/dotfiles\"..."), "type": Number(3)}) })))
2023-03-02T21:29:55.394 helix_term::application [INFO] window/logMessage: LogMessageParams { typ: Info, message: "20:29:55.388 INFO BackgroundAnalysis: resolving glob \"**/*@(.sh|.inc|.bash|.command)\" inside \"file:///home/fenris/dotfiles\"..." }
2023-03-02T21:29:55.411 helix_lsp::transport [INFO] <- {"jsonrpc":"2.0","method":"window/logMessage","params":{"type":3,"message":"20:29:55.411 INFO BackgroundAnalysis: Glob resolved with 22 files after 0.022 seconds"}}
2023-03-02T21:29:55.411 helix_term::application [DEBUG] received editor event: LanguageServerMessage((0, Notification(Notification { jsonrpc: Some(V2), method: "window/logMessage", params: Map({"message": String("20:29:55.411 INFO BackgroundAnalysis: Glob resolved with 22 files after 0.022 seconds"), "type": Number(3)}) })))
2023-03-02T21:29:55.411 helix_term::application [INFO] window/logMessage: LogMessageParams { typ: Info, message: "20:29:55.411 INFO BackgroundAnalysis: Glob resolved with 22 files after 0.022 seconds" }
2023-03-02T21:29:55.412 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:29:55.425 helix_lsp::transport [INFO] <- {"jsonrpc":"2.0","method":"window/logMessage","params":{"type":3,"message":"20:29:55.425 INFO BackgroundAnalysis: Completed after 0.036 seconds."}}
2023-03-02T21:29:55.425 helix_term::application [DEBUG] received editor event: LanguageServerMessage((0, Notification(Notification { jsonrpc: Some(V2), method: "window/logMessage", params: Map({"message": String("20:29:55.425 INFO BackgroundAnalysis: Completed after 0.036 seconds."), "type": Number(3)}) })))
2023-03-02T21:29:55.425 helix_term::application [INFO] window/logMessage: LogMessageParams { typ: Info, message: "20:29:55.425 INFO BackgroundAnalysis: Completed after 0.036 seconds." }
2023-03-02T21:29:55.514 helix_term::application [DEBUG] received editor event: IdleTimer
2023-03-02T21:29:55.895 helix_lsp::transport [INFO] <- {"jsonrpc":"2.0","method":"window/logMessage","params":{"type":2,"message":"20:29:55.894 WARNING ⛔️ ShellCheck: disabling linting as no executable was found at path 'shellcheck'"}}
2023-03-02T21:29:55.895 helix_term::application [DEBUG] received editor event: LanguageServerMessage((0, Notification(Notification { jsonrpc: Some(V2), method: "window/logMessage", params: Map({"message": String("20:29:55.894 WARNING ⛔\u{fe0f} ShellCheck: disabling linting as no executable was found at path 'shellcheck'"), "type": Number(2)}) })))
2023-03-02T21:29:55.895 helix_term::application [INFO] window/logMessage: LogMessageParams { typ: Warning, message: "20:29:55.894 WARNING ⛔\u{fe0f} ShellCheck: disabling linting as no executable was found at path 'shellcheck'" }
2023-03-02T21:29:55.895 helix_lsp::transport [INFO] <- {"jsonrpc":"2.0","method":"textDocument/publishDiagnostics","params":{"uri":"file:///home/fenris/dotfiles/bin/password.sh","version":0,"diagnostics":[]}}
2023-03-02T21:29:55.895 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:29:55.895 helix_term::application [DEBUG] received editor event: LanguageServerMessage((0, Notification(Notification { jsonrpc: Some(V2), method: "textDocument/publishDiagnostics", params: Map({"diagnostics": Array [], "uri": String("file:///home/fenris/dotfiles/bin/password.sh"), "version": Number(0)}) })))
2023-03-02T21:29:59.256 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:29:59.536 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:29:59.536 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:29:59.937 helix_term::application [DEBUG] received editor event: IdleTimer
2023-03-02T21:30:00.056 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:00.056 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:00.157 globset [DEBUG] glob converted to regex: Glob { glob: "*.{zip,gz,bz2,zst,lzo,sz,tgz,tbz2,lz,lz4,lzma,lzo,z,Z,xz,7z,rar,cab}", re: "(?-u)^[^/]*\\.(cab|rar|7z|xz|Z|z|lzo|lzma|lz4|lz|tbz2|tgz|sz|lzo|zst|bz2|gz|zip)$", opts: GlobOptions { case_insensitive: false, literal_separator: true, backslash_escape: true }, tokens: Tokens([ZeroOrMore, Literal('.'), Alternates([Tokens([Literal('c'), Literal('a'), Literal('b')]), Tokens([Literal('r'), Literal('a'), Literal('r')]), Tokens([Literal('7'), Literal('z')]), Tokens([Literal('x'), Literal('z')]), Tokens([Literal('Z')]), Tokens([Literal('z')]), Tokens([Literal('l'), Literal('z'), Literal('o')]), Tokens([Literal('l'), Literal('z'), Literal('m'), Literal('a')]), Tokens([Literal('l'), Literal('z'), Literal('4')]), Tokens([Literal('l'), Literal('z')]), Tokens([Literal('t'), Literal('b'), Literal('z'), Literal('2')]), Tokens([Literal('t'), Literal('g'), Literal('z')]), Tokens([Literal('s'), Literal('z')]), Tokens([Literal('l'), Literal('z'), Literal('o')]), Tokens([Literal('z'), Literal('s'), Literal('t')]), Tokens([Literal('b'), Literal('z'), Literal('2')]), Tokens([Literal('g'), Literal('z')]), Tokens([Literal('z'), Literal('i'), Literal('p')])])]) }
2023-03-02T21:30:00.157 globset [DEBUG] built glob set; 0 literals, 0 basenames, 0 extensions, 0 prefixes, 0 suffixes, 0 required extensions, 1 regexes
2023-03-02T21:30:00.157 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/.git: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:00.158 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/roles/paru/.travis.yml: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:00.160 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/roles/nginx_reverse_proxy/templates/.env: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:00.160 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/roles/arch_base/templates/.gitconfig: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:00.160 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/roles/arch_base/.travis.yml: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:00.161 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/roles/podman/.travis.yml: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:00.161 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/roles/homeassistant/.travis.yml: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:00.161 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/roles/helix/.travis.yml: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:00.161 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/vscodium/.config: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:00.161 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/rofi/.config: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:00.162 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/bat/.config: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:00.162 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/zsh/.config: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:00.162 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/zsh/.zshrc: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:00.162 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/gtk/.config: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:00.162 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/gtk/.gtkrc-2.0: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:00.162 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/qt/.config: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:00.162 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/git/.gitconfig: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:00.162 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/wm/.config: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:00.162 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/helix/.config: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:00.162 helix_term::ui [DEBUG] file_picker init 5.231676ms
2023-03-02T21:30:00.162 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:00.162 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:00.499 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:00.499 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:00.663 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:00.663 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:00.813 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:00.813 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:00.964 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:00.964 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:01.113 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:01.113 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:01.513 helix_term::application [DEBUG] received editor event: IdleTimer
2023-03-02T21:30:01.514 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:01.514 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:01.953 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:01.953 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:02.253 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:02.253 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:02.253 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:02.655 helix_term::application [DEBUG] received editor event: IdleTimer
2023-03-02T21:30:02.980 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:02.980 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:03.381 helix_term::application [DEBUG] received editor event: IdleTimer
2023-03-02T21:30:04.000 globset [DEBUG] glob converted to regex: Glob { glob: "*.{zip,gz,bz2,zst,lzo,sz,tgz,tbz2,lz,lz4,lzma,lzo,z,Z,xz,7z,rar,cab}", re: "(?-u)^[^/]*\\.(cab|rar|7z|xz|Z|z|lzo|lzma|lz4|lz|tbz2|tgz|sz|lzo|zst|bz2|gz|zip)$", opts: GlobOptions { case_insensitive: false, literal_separator: true, backslash_escape: true }, tokens: Tokens([ZeroOrMore, Literal('.'), Alternates([Tokens([Literal('c'), Literal('a'), Literal('b')]), Tokens([Literal('r'), Literal('a'), Literal('r')]), Tokens([Literal('7'), Literal('z')]), Tokens([Literal('x'), Literal('z')]), Tokens([Literal('Z')]), Tokens([Literal('z')]), Tokens([Literal('l'), Literal('z'), Literal('o')]), Tokens([Literal('l'), Literal('z'), Literal('m'), Literal('a')]), Tokens([Literal('l'), Literal('z'), Literal('4')]), Tokens([Literal('l'), Literal('z')]), Tokens([Literal('t'), Literal('b'), Literal('z'), Literal('2')]), Tokens([Literal('t'), Literal('g'), Literal('z')]), Tokens([Literal('s'), Literal('z')]), Tokens([Literal('l'), Literal('z'), Literal('o')]), Tokens([Literal('z'), Literal('s'), Literal('t')]), Tokens([Literal('b'), Literal('z'), Literal('2')]), Tokens([Literal('g'), Literal('z')]), Tokens([Literal('z'), Literal('i'), Literal('p')])])]) }
2023-03-02T21:30:04.000 globset [DEBUG] built glob set; 0 literals, 0 basenames, 0 extensions, 0 prefixes, 0 suffixes, 0 required extensions, 1 regexes
2023-03-02T21:30:04.000 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/.git: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:04.001 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/roles/paru/.travis.yml: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:04.003 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/roles/nginx_reverse_proxy/templates/.env: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:04.003 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/roles/arch_base/templates/.gitconfig: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:04.003 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/roles/arch_base/.travis.yml: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:04.004 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/roles/podman/.travis.yml: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:04.004 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/roles/homeassistant/.travis.yml: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:04.004 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/roles/helix/.travis.yml: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:04.005 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/vscodium/.config: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:04.005 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/rofi/.config: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:04.005 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/bat/.config: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:04.005 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/zsh/.config: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:04.005 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/zsh/.zshrc: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:04.005 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/gtk/.config: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:04.005 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/gtk/.gtkrc-2.0: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:04.005 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/qt/.config: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:04.005 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/git/.gitconfig: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:04.005 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/wm/.config: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:04.005 ignore::walk [DEBUG] ignoring /home/fenris/dotfiles/dotfiles_old/helix/.config: Ignore(IgnoreMatch(Hidden))
2023-03-02T21:30:04.005 helix_term::ui [DEBUG] file_picker init 5.224116ms
2023-03-02T21:30:04.005 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:04.005 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:04.401 helix_term::application [DEBUG] received editor event: IdleTimer
2023-03-02T21:30:04.402 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:04.402 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:04.563 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:04.563 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:04.964 helix_term::application [DEBUG] received editor event: IdleTimer
2023-03-02T21:30:04.964 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:04.964 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:05.163 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:05.163 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:05.203 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:05.203 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:05.243 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:05.243 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:05.283 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:05.283 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:05.324 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:05.324 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:05.364 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:05.364 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:05.764 helix_term::application [DEBUG] received editor event: IdleTimer
2023-03-02T21:30:05.797 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:05.797 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:06.664 helix_view::editor [ERROR] Failed to initialize the LSP for `source.md` { cannot find binary path }
2023-03-02T21:30:06.669 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:06.669 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:06.670 helix_view::document [DEBUG] id 3 modified - last saved: 0, current: 0
2023-03-02T21:30:07.065 helix_term::application [DEBUG] received editor event: IdleTimer
2023-03-02T21:30:09.458 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:09.459 helix_view::document [DEBUG] id 3 modified - last saved: 0, current: 0
2023-03-02T21:30:10.473 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:10.474 helix_view::document [DEBUG] id 3 modified - last saved: 0, current: 0
2023-03-02T21:30:10.874 helix_term::application [DEBUG] received editor event: IdleTimer
2023-03-02T21:30:11.202 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:11.203 helix_view::document [DEBUG] id 3 modified - last saved: 0, current: 0
2023-03-02T21:30:11.328 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:11.329 helix_view::document [DEBUG] id 3 modified - last saved: 0, current: 0
2023-03-02T21:30:11.403 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:11.404 helix_view::document [DEBUG] id 3 modified - last saved: 0, current: 0
2023-03-02T21:30:11.824 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:11.825 helix_view::document [DEBUG] id 3 modified - last saved: 0, current: 0
2023-03-02T21:30:12.929 helix_view::document [DEBUG] id 2 modified - last saved: 0, current: 0
2023-03-02T21:30:12.929 helix_view::document [DEBUG] id 3 modified - last saved: 0, current: 0

</details>

### Platform

Linux

### Terminal Emulator

Alacritty 0.11.0

### Helix Version

helix 22.12

**Comment by pascalkuthe** [comment]
Thanks for the detailed bug report! This is a duplicate of #5632. A fix is currently blocked on https://github.com/helix-editor/helix/pull/5581.
